src/map/utils/charutils.cpp

```diff
@@ -672,20 +672,17 @@ namespace charutils
             PChar->profile.mhflag = (uint16)rset->getInt("mhflag");
             PChar->profile.title  = (uint16)rset->getInt("title");
 
-            // TODO:
-            /*
             std::array<uint8, 512> bazaarMessageArray;
             db::extractFromBlob(rset, "bazaar_message", bazaarMessageArray);
             const char* bazaarMessageStr = reinterpret_cast<const char*>(bazaarMessageArray.data());
             if (bazaarMessageStr != nullptr)
             {
-                PChar->bazaar.message.insert(0, (char*)_sql->GetData(7));
+                PChar->bazaar.message.insert(0, bazaarMessageStr);
```

Comment:
I'm not sure why it used Insert to begin with, probably not necessary, but approving anyway.

Reply:
Yeah, I hate this pattern, but it's been there forever so it's probably OK